templates/navigation/menu--main.html.twig

```diff
@@ -28,22 +28,28 @@
 {% macro menu_links(items, attributes, menu_level) %}
   {% import _self as menus %}
   {% if items %}
+    {# When outputting menu level as a class, use the "cosmetic" value so that
+       classnames begin with 1 instead of 0. #}
+    {% set menu_level_cosmetic = menu_level + 1 %}
+    {% set component = 'cd-nav-level-' ~ menu_level_cosmetic %}
 
     {%
       set menu_classes = [
         'menu',
-        menu_level > 0 ? 'cd-main-menu__dropdown',
+        'cd-nav__menu--level-' ~ menu_level_cosmetic,
+        menu_level > 0 ? component ~ '__dropdown',
```

Comment:
@rupl I set up a branch to deploy to Feature site for VRT, and discover the steps where sass is still attempted to be compiled. It also served to run the e2e tests (which I couldn't get working locally) See https://github.com/UN-OCHA/common-design-site/pull/250
I think the highlighted code above breaks the tests `MainMenuDropdown` as it changes the class the test looks for. This means any sub themes with tests copied from the base theme will have to be updated too.

Reply:
Ah yeah, right... I had remembered a couple tests but the main nav ended up taking several tries and I let the testing slip. Thanks for catching it! Thanks for setting up a better v8 PR too.

Reply:
Fixed by f85cd50